assets/js/components/Badge.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Badge component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+const Badge = ( { label } ) => (
+	<span className="googlesitekit-badge">{ label }</span>
```

Comment:
The IB mentions using `googlesitekit__badge`, but I think `googlesitekit-badge` _is_ the more appropriate CSS class name. Just flagging that here that it's been noticed but I think this is better. 🙂